src/test/java/org/salesforce/runner/RunTests.java

```diff
@@ -23,22 +23,21 @@
 import salesforce.api.entities.Contact;
 import salesforce.api.entities.PriceBook;
 import salesforce.api.entities.Response;
-import static salesforce.api.Authentication.token;
-import java.util.Properties;
 
 @CucumberOptions(
         features = {"src/test/resources/features/"},
         plugin = {"html:build/cucumber/cucumber-pretty.html", "json:build/cucumber/cucumber.json"},
         glue = {"org.salesforce"}
 )
 public class RunTests extends AbstractTestNGCucumberTests {
+
```

Comment:
Delete  blank line

Reply:
According to the conventions we have that space after all our classes.